utils/src/deployer.rs

```diff
@@ -472,6 +625,30 @@ pub async fn is_proxy_contract(
     Ok(implementation_address != H160::zero())
 }
 
+pub async fn is_valid_admin_light_client_proxy<M: Middleware + 'static>(
```

Comment:
Since the implementation of the function is the same and we only call the `owner` function of (I think) `OwnableUpgradable` we could create the bindings for `OwnableUpgradable` and instantiate that rust contract type. Then we can use the same function for all our proxy contracts and don't have to duplicate the function implementation.

Reply:
great call https://github.com/EspressoSystems/espresso-sequencer/commit/e44b49f5e7b2565ea34a974d2646cf2d5f9ee7c0